test/index.test.js

```diff
@@ -1889,13 +1892,13 @@ describe('index', function () {
             headers['x-event-key'] = 'issue:created';
 
             return scm.canHandleWebhook(headers, testPayloadPush).then(result => {
-                assert.isFalse(result);
+                assert.isTrue(result);
```

Comment:
Can you update test title from `false` to `true`

Reply:
Thank you. I fixed it 👍